src/GlobalState.js

```diff
@@ -32,6 +32,7 @@ class Provider extends Component {
     currentModal: null,
     auth: LocalStorage.getItem(AUTH) || {},
     networkState: {},
+    searchTerm: ''
```

Comment:
Why have we shifted search into global state? Global state is only for stuff that applies across the entire app. The search should be contained within the single event page component tree

Reply:
This line was duplicated into 3 different components so I thought it's better to move to global state
```
  state = {
    search: ''
  }
   handleSearch = event => {
    this.setState({
      search: event.target.value
    })
  }
```

Reply:
I think there is redundant code in the components, that's why it looks like things are being duplicated all over. I would say we should search to be contained with `EventParticipants`.